2016/proposals/singh-shubham.md

```diff
@@ -0,0 +1,210 @@
+# Manage workflow for instructor training
+
+## Abstract
+
+Computing Skills have now become essential more so in the field of Science, but many a time we find researchers struggling with code or tools to get things done.
+Software Carpentary is a step in this direction.
+Since 1998, Software Carpentry has been teaching researchers in science, engineering, medicine, and related disciplines the computing skills they need to get more done in less time and with less pain.
+Software Carpentary , uses AMY ( a django based web application) to to keep track of who's qualified to teach, what workshops have run, who taught and attended. As more and more people are applying to become instructors at Software Carpentary. It has become essential to manage the workflow for instructor training.
+
+This project aims at designing such a workflow and implementing it by adding appropriate views to AMY.
```

Comment:
"This project will design such a workflow and implement it by adding appropriate views to AMY." (As Yoda might have said, do or do not, there is no "aims".)